hashtagsv2/hashtags/templates/hashtags/graph.html

```diff
@@ -0,0 +1,199 @@
+                    {% endfor %}
+                </h2>
+                <h2>Statistics</h2>
+            </div>
+            <div class="one-third column">
+                <p class="stats-date-range">{{ oldest }} - {{ newest }}</p>
+                <table class="stats-table">
+                    <tr>
+                        <td class="stat">{{ revisions }}</td>
+                        <td class="stat-label">revision{{ revisions|pluralize:"s" }}</td>
+                    </tr>
+                    <tr>
+                        <td class="stat">{{ pages }}</td> 
+                        <td class="stat-label">page{{ pages|pluralize:"s" }}</td>
+                    </tr>
+                    <tr>
+                        <td class="stat">{{ users }}</td>
+                        <td class="stat-label">user{{ users|pluralize:"s" }}</td>
+                    </tr>
+                    <tr>
+                        <td class="stat">{{ projects }}</td>
+                        <td class="stat-label">project{{ projects|pluralize:"s" }}</td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+    </div>
+    <hr>
+    <div class="row"><h1>Top projects</h1></div>
+    <div class="row">
+        <div class="two-thirds column">
+            <canvas id="projectsChart" width="400" height="400"></canvas>
+        </div>
+        <div class="one-third column">
+            <a href="/projects_csv/?{{ query_string }}"><button>Download CSV</button></a>
+            <a href="/all_projects/?{{ query_string }}"><button>View Full Stats</button></a>
+        </div>
+    </div>
+    <hr>
+    <div class="row"><h1>Top users</h1></div>
+    <div class="row">
+        <div class="two-thirds column">
+            <canvas id="usersChart" width="400" height="400"></canvas>
+        </div>
+        <div class="one-third column">
+            <a href="/users_csv/?{{ query_string }}"><button>Download CSV</button></a>
+            <a href="/all_users/?{{ query_string }}"><button>View Full Stats</button></a>    
+        </div>
+    </div>
+    <hr>
+    <div class="row"><h1>Edits over time</h1></div>
+    <div class="row">
+        <div>
+            <canvas id="timeChart" width="400" height="400"></canvas>
+        </div>
+        <div class="row">
+            <a href="/time_csv/?{{ query_string }}"><button>Download CSV</button></a>
+        </div>
+    </div>
+    <hr>
+    <script> var url_param = '{{ query_string }}'; </script>
+    <script>
+        $.ajax({
+            url: '/api/top_project_stats/?' + url_param,
+            dataType: 'json',
+            success: function(data){
+                var ctx = document.getElementById('projectsChart');
+                var projectsChart = new Chart(ctx, {
+                    type: 'bar',
+                    data: {
+                        labels: data['projects'],
+                        datasets: [{
+                            label: 'number of Edits',
```

Comment:
Can we fix the capitalisation to "Number of edits"? + I think this is the same on each of the other graphs too.